src/renderer/components/path-chooser.tsx

```diff
@@ -1,20 +1,30 @@
 import { remote } from 'electron';
-import { AppState, appState } from '../state';
-import { writeReport } from '../../report';
+import { appState } from '../state';
 
-export async function chooseFolder() {
+export function chooseFolder() {
   const currentWindow = remote.BrowserWindow.getAllWindows()[0];
-
-  remote.dialog.showOpenDialog(
+  
+  let filePaths = remote.dialog.showOpenDialog(
```

Comment:
```suggestion
  const filePaths = remote.dialog.showOpenDialog(
```

Also, `showOpenDialog` is an asynchronous method, so you'll have to `await` it. Otherwise, you'll set `filePaths` to the `Promise` rather than the actual result.